xml/ha_storage_protection.xml

```diff
@@ -842,21 +793,128 @@ SBD_WATCHDOG_TIMEOUT=5</screen>
+         scenarios for two-node clusters.
+      </para>
+   </important>
+
+   <procedure xml:id="pro.ha.storage.protect.confdiskless">
+    <title>Configuring Diskless SBD</title>
+    <step>
+     <para>Open the file <filename>/etc/sysconfig/sbd</filename> and use
+      the following entries:</para>
+     <screen>SBD_PACEMAKER=yes
+SBD_STARTMODE=always
+SBD_DELAY_START=no
+SBD_WATCHDOG_DEV=/dev/watchdog
+SBD_WATCHDOG_TIMEOUT=5</screen>
+      <para>
+       The <varname>SBD_DEVICE</varname> entry is not needed as no shared
+       disk is used. When this parameter is missing, the <systemitem>sbd</systemitem>
+       service does not start any watcher process for SBD devices.
+      </para>
+    </step>
+    <step>
+     <para>On each node, enable the SBD service to start at boot time: </para>
+     <screen>&prompt.root;<command>systemctl</command> enable sbd</screen>
+    </step>
+    <step>
+     <para>Restart the cluster stack on each node to ensure that the
+      <systemitem>sbd</systemitem> service is started:</para>
+     <screen>&prompt.root;<command>systemctl</command> stop pacemaker
+&prompt.root;<command>systemctl</command> start pacemaker</screen>
+     <para> This automatically triggers the start of the SBD daemon. </para>
+    </step>
+    <step>
+      <para>
+       Check if the parameter <parameter>have-watchdog=true</parameter> has
+       been automatically set:
+      </para>
+      <screen>&prompt.root;<command>crm</command> configure show | grep have-watchdog
+         have-watchdog=true</screen>
+    </step>
+    <step>
+     <para>Run <command>crm configure</command> and set the following cluster
+      properties on the &crmshell;:</para>
+<screen>&prompt.crm.conf;<command>property</command> stonith-enabled="true" <xref
+        linkend="co.ha.sbd.st.enabled" xrefstyle="select:label"/>
+&prompt.crm.conf;<command>property</command> stonith-watchdog-timeout=5 <co
+        xml:id="co.ha.sbd.diskless.watchdog-timeout"/></screen>
+    <calloutlist>
+     <callout arearefs="co.ha.sbd.st.enabled">
+      <para>
+       This is the default configuration, because clusters with &stonith; are not supported.
+       But in case &stonith; had been deactivated for testing purposes,
+       make sure this parameter is set to <literal>true</literal>.</para>
+     </callout>
+     <callout arearefs="co.ha.sbd.watchdog-timeout">
+      <para>For diskless SBD, this parameter must not equal zero.
+       It defines after how long it is assumed that the fencing target has already
+       self-fenced. Therefore its value needs to be &gt;= the value of
+       <varname>SBD_WATCHDOG_TIMEOUT</varname> in <filename>/etc/sysconfig/sbd</filename>.
+       Starting with &productname; 15, if you set it to a negative value, Pacemaker
+       will automatically calculate an appropriate timeout based on the value of
+       <parameter>SBD_WATCHDOG_TIMEOUT</parameter> that is configured in
+       <filename>/etc/sysconfig/sbd</filename>.
```

Comment:
Or to be precise, twice of SBD_WATCHDOG_TIMEOUT as also described in pacemaker/doc/Pacemaker_Explained/en-US/Ch-Options.txt:
https://github.com/ClusterLabs/pacemaker/commit/cbf53fe834377476d9edd713d4bd94998d938ee4

Reply:
Great, added this information.